pandas/io/stata.py

```diff
@@ -160,15 +160,29 @@
 
 Examples
 --------
+
+Creating a dummy stata for this example
+>>>df = pd.DataFrame({'animal': ['falcon', 'parrot', 'falcon',
```

Comment:
You must escape the `{` here since this is an f-string.  You need to use `{{` whenever you want a `{` in code.